app/src/main/res/xml/preferences.xml

```diff
@@ -406,5 +406,20 @@
 
     </PreferenceCategory>
 
+    <PreferenceCategory
+        android:key="preference_category_online"
+        android:title="@string/preference_category_online"
+        >
+
+        <!--  if we move this to another PreferenceGroup, we should update code for the findPreference() call -->
+        <SwitchPreference
+            android:key="preference_use_camera2"
+            android:title="@string/preference_use_camera2"
+            android:summary="@string/preference_use_camera2_summary"
+            android:defaultValue="false"
+            />
```

Comment:
Please follow proper formatting (look at other preferences and how the brackets are closed).